prac2-pages/anshumali_suri/lib/main.dart

```diff
@@ -0,0 +1,202 @@
+                    decoration: InputDecoration(
+                      border: OutlineInputBorder(
+                        borderRadius: BorderRadius.circular(10.0),
+                      ),
+                      labelText: 'Enter you username',
+                    ),
+                  ),
+
+                ],
+              ),
+            ),
+            SizedBox(
+              height: 20,
+            ),
+            Container(
+              padding: const EdgeInsets.fromLTRB(10, 10, 10, 0),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children:<Widget>[
+                  Text(
+                    'Password',
+
+                    style: TextStyle(
+                      fontSize: 25,
+                      fontWeight: FontWeight.bold,
+                    ),
+                  ),
+                  SizedBox(
+                    height: 5,
+                  ),
+                  TextField(
+                    obscureText: true,
+                    controller: passwordController,
+                    decoration:  InputDecoration(
+                      border: OutlineInputBorder(
+                        borderRadius: BorderRadius.circular(10.0),
+                      ),
+                      labelText: 'Enter your password',
+                    ),
+                  ),
+
+                ],
+              ),
+            ),
+            SizedBox(
+              height: 30,
+            ),
+
+            Container(
+              height: 65,
+              padding: const EdgeInsets.fromLTRB(10, 0, 10, 0),
+              child: ElevatedButton(
+                child: const Text('Log In',
+                  style: TextStyle(
+                    fontSize: 16,
+
+                  ),
+                ),
+                style: ElevatedButton.styleFrom(
+                  primary: Colors.grey[800],
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(10),
+                  ),
+                ),
+                onPressed: () {
+                  print(nameController.text);
+                  print(passwordController.text);
+                },
+              ),
+            ),
+
+            SizedBox(
+              height: 20,
+            ),
+
+            Container(
+
+              alignment: Alignment.center,
+              child: CheckboxListTile(
+
+                activeColor: Colors.grey[800],
+                controlAffinity: ListTileControlAffinity.leading,
+                title: Transform.translate(
+                  offset: const Offset(-20, 0),
+                  child: Text(
+                    'Keep me logged in',
+                    style: TextStyle(
+
+                      fontSize: 20,
+                    ),
+                  ),
+                ),
+                value: value,
+                onChanged: (value) => setState(() => this.value = value!),
```

Comment:
value! tells the compiler that it is null-safe. I believe you wanted to use the logical NOT, so please look into this as well..

Reply:
I deliberately used null safety operator and did not wanted to use not operator.
<img width="846" alt="Screenshot 2022-12-21 at 1 12 30 PM" src="https://user-images.githubusercontent.com/114004856/208848719-30ddf84f-5091-4a55-8d07-20bd69835d32.png">